core/src/test/scala/unit/kafka/log/LogCleanerManagerTest.scala

```diff
@@ -526,6 +560,16 @@ class LogCleanerManagerTest extends Logging {
       logDirFailureChannel = new LogDirFailureChannel(10))
   }
 
+  private def createLogConfig(segmentSize: Int, cleanupPolicy: String): LogConfig = {
```

Comment:
nit: I'd suggest a name which suggests the configuration. Maybe `createLowRetentionLogConfig` or something like that?